jvm-system/src/main/scala/com/avast/server/toolkit/system/SystemModule.scala

```diff
@@ -0,0 +1,62 @@
+package com.avast.server.toolkit.system
+
+import java.io.{OutputStream, Reader}
+import java.security.SecureRandom
+
+import cats.effect.{Clock, Sync}
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import com.avast.server.toolkit.system.console.Console
+import com.avast.server.toolkit.system.random.Random
+
+import scala.language.higherKinds
+import scala.{Console => SConsole}
+
+/** Provides JVM system services such as standard in/out, time and random number generation.
+  *
+  * Contains already initialized standard [[com.avast.server.toolkit.system.console.Console]], default [[cats.effect.Clock]]
+  * and [[com.avast.server.toolkit.system.random.Random]] or you can use `make*` factory methods to create more or customized instances.
+  */
+class SystemModule[F[_]: Sync](val console: Console[F], val clock: Clock[F], val random: Random[F]) {
+
+  /** Makes [[Console]] with the provided input and output streams. */
+  def makeConsole(in: Reader, out: OutputStream): F[Console[F]] = SystemModule.makeConsole(in, out)
+
+  /** Makes [[Random]] with default random seed. */
+  def makeRandom: F[Random[F]] = SystemModule.makeRandom
+
+  /** Makes [[Random]] with the provided `seed`. */
+  def makeRandom(seed: Long): F[Random[F]] = SystemModule.makeRandom(seed)
+
+  /** Makes [[Random]] based on [[java.security.SecureRandom]] with default random seed. */
+  def makeSecureRandom: F[Random[F]] = SystemModule.makeSecureRandom
```

Comment:
is there ever a case for creating Random instead of SecureRandom? Would it make sense to only have builder for SecureRandom?

Reply:
honestly, idk ... let's keep them separate then